src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Xproj/GlobalJsonRemover.cs

```diff
@@ -74,8 +74,10 @@ internal GlobalJsonRemover(IServiceProvider serviceProvider, IFileSystem fileSys
         public int OnAfterOpenSolution(object pUnkReserved, int fNewSolution)
         {
             UIThreadHelper.VerifyOnUIThread();
+#pragma warning disable RS0030 // Do not used banned APIs
             DTE2 dte = _serviceProvider.GetService<DTE2, DTE>();
```

Comment:
Do we want to eventually refactor this someday or do we feel this usage is acceptable?

Reply:
I guess I am actually asking: do we want to file bugs for some of these cases to refactor them later?

Reply:
Tracked by https://github.com/dotnet/project-system/issues/4375, usually I'd baseline them in GlobalSupressions but just following existing usage in that bug.